packages/ecs-agent/0003-bottlerocket-remove-unsupported-capabilities.patch

```diff
@@ -52,8 +52,8 @@ index 09e01825..a7c36cfd 100644
 +	//	capabilities = agent.appendEFSVolumePluginCapabilities(capabilities, cap)
 +	// }
  
- 	return capabilities, nil
- }
+ 	// support fsxWindowsFileServer on ecs capabilities
+ 	capabilities = agent.appendFSxWindowsFileServerCapabilities(capabilities)
```

Comment:
Does this end up registering a capability even though we're not running Windows?

Reply:
I'm not sure yet, I am building an image to launch & see what'll happen as it comes up.

Reply:
I don't see the capability listed on a launched instance. The [`appendFSxWindowsFileServerCapabilities` function is a noop for the Linux build](https://github.com/aws/amazon-ecs-agent/blob/1489adfaec05c559ea4e1490774d1f86261cf936/agent/app/agent_capability_unix.go#L186-L188) so the FSx capability is not registered.

```json
{
  "bottlerocket.variant": "aws-ecs-1",
  "bottlerocket.version": "1.0.2",
  "build-commit": "58147e894dc841a65d1f15bfea940ca026f2222c",
  "com.amazonaws.ecs.capability.docker-remote-api.1.17": null,
  "com.amazonaws.ecs.capability.docker-remote-api.1.18": null,
  "com.amazonaws.ecs.capability.docker-remote-api.1.19": null,
  "com.amazonaws.ecs.capability.docker-remote-api.1.20": null,
  "com.amazonaws.ecs.capability.docker-remote-api.1.21": null,
  "com.amazonaws.ecs.capability.docker-remote-api.1.22": null,
  "com.amazonaws.ecs.capability.docker-remote-api.1.23": null,
  "com.amazonaws.ecs.capability.docker-remote-api.1.24": null,
  "com.amazonaws.ecs.capability.docker-remote-api.1.25": null,
  "com.amazonaws.ecs.capability.docker-remote-api.1.26": null,
  "com.amazonaws.ecs.capability.docker-remote-api.1.27": null,
  "com.amazonaws.ecs.capability.docker-remote-api.1.28": null,
  "com.amazonaws.ecs.capability.docker-remote-api.1.29": null,
  "com.amazonaws.ecs.capability.docker-remote-api.1.30": null,
  "com.amazonaws.ecs.capability.docker-remote-api.1.31": null,
  "com.amazonaws.ecs.capability.docker-remote-api.1.32": null,
  "com.amazonaws.ecs.capability.ecr-auth": null,
  "com.amazonaws.ecs.capability.logging-driver.awslogs": null,
  "com.amazonaws.ecs.capability.logging-driver.json-file": null,
  "com.amazonaws.ecs.capability.logging-driver.none": null,
  "com.amazonaws.ecs.capability.selinux": null,
  "com.amazonaws.ecs.capability.task-iam-role": null,
  "com.amazonaws.ecs.capability.task-iam-role-network-host": null,
  "ecs.ami-id": "ami-050784c46332663c0",
  "ecs.availability-zone": "us-west-2d",
  "ecs.capability.container-health-check": null,
  "ecs.capability.container-ordering": null,
  "ecs.capability.docker-plugin.local": null,
  "ecs.capability.ecr-endpoint": null,
  "ecs.capability.efs": null,
  "ecs.capability.env-files.s3": null,
  "ecs.capability.execution-role-awslogs": null,
  "ecs.capability.execution-role-ecr-pull": null,
  "ecs.capability.full-sync": null,
  "ecs.capability.pid-ipc-namespace-sharing": null,
  "ecs.capability.private-registry-authentication.secretsmanager": null,
  "ecs.capability.secrets.asm.bootstrap.log-driver": null,
  "ecs.capability.secrets.asm.environment-variables": null,
  "ecs.capability.secrets.ssm.bootstrap.log-driver": null,
  "ecs.capability.secrets.ssm.environment-variables": null,
  "ecs.capability.task-cpu-mem-limit": null,
  "ecs.cpu-architecture": "x86_64",
  "ecs.instance-type": "t3.xlarge",
  "ecs.os-type": "linux"
}
```